packages/blade/src/components/Icons/_Svg/Svg/types.ts

```diff
@@ -1,7 +1,9 @@
+import type { StyledProps } from '~utils';
+
 export type SvgProps = {
   children: React.ReactNode;
   fill?: string;
-  height: string;
+  height: `${string}px`;
   viewBox: string;
-  width: string;
-};
+  width: `${string}px`;
```

Comment:
Should we restrict this to `px` value only? Unitless values would work same way right?

Reply:
reverted this change. https://github.com/razorpay/blade/pull/1016#discussion_r1121226679